docs/pix2struct-chyrons/v1.0/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Pix2struct Chyrons",
+  "description": "extracts text from input timeframes based on chyron queries using the pix2struct Doc-VQA model",
+  "app_version": "v1.0",
+  "mmif_version": "1.0.0",
+  "analyzer_version": "1",
+  "app_license": "MIT",
+  "analyzer_license": "apache-2.0",
+  "identifier": "http://apps.clams.ai/pix2struct-chyrons/v1.0",
+  "url": "https://github.com/clamsproject/app-pix2struct-docvqa-wrapper",
+  "input": [
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/VideoDocument/v1",
+      "required": true
+    }
+  ],
+  "output": [
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/TextDocument/v1",
+      "properties": {
+        "description": "extracted text from input timeframes based on chyron queries using the pix2struct Doc-VQA model"
+      }
+    },
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/Alignment/v1",
```

Comment:
https://github.com/clamsproject/app-pix2struct-chyrons/blob/cd959fb8b7954769de590924670cdb53e9c475c0/app.py#L49 says the newly created view will have `Relation` but shouldn't it be `Alignment`?